lists-service/src/main/java/au/org/ala/listsapi/controller/AuthUtils.java

```diff
@@ -1,7 +1,19 @@
-package au.org.ala.listsapi.controller;
```

Comment:
Moved and renamed file -> was MongoUtils in controller dir. Was misleading as had lots of ES code.